src/types/onyx/WalletStatement.ts

```diff
@@ -1,6 +1,11 @@
 type WalletStatement = {
     /** Whether we are currently generating a PDF version of the statement */
     isGenerating: boolean;
+
+    /** 
+      yearMonth - key with filename as value, boolean value added for isGenerating key
+     */
+    [yearMonth: string]: string | boolean;
 };
```

Comment:
@pasyukevich Could you help to explain more why we added a boolean type here? I see that we only use this value for filename `const fileName = walletStatement[yearMonth];`

Reply:
Boolean was added to be compatible with `isGenerating` field in the `WalletStatement` type